odoo/cli/populate.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+
+import time
+import fnmatch
+import logging
+import optparse
+import odoo
+
+from . import Command
+_logger = logging.getLogger(__name__)
+
+
+class Populate(Command):
+
+    def run(self, cmdargs):
+        parser = odoo.tools.config.parser
+        group = optparse.OptionGroup(parser, "Populate Configuration")
+        group.add_option("--populate-level", dest="populate_level",
+                    help="Populate database with auto-generated data. Value should be the population size: low, medium or high",
+                    my_default='low')
```

Comment:
`my_default` is an odoo specific workaround
https://github.com/odoo/odoo/blob/f2b9bdf7b2b2f6fdad2486adbb7afaa58d4ad0e4/odoo/tools/config.py#L36

I think that you can use `default`